x/oracle/types/ballot.go

```diff
@@ -203,3 +204,12 @@ func ClaimMapToSlices(claims map[string]Claim, rewardMap map[string]bool) ([]Cla
 	})
 	return c, r
 }
+
+func rewardSetToMap(rewardSet []string) map[string]bool {
+	rewardMap := make(map[string]bool)
+	for _, v := range rewardSet {
+		rewardMap[v] = true
+	}
+
+	return rewardMap
+}
```

Comment:
Since we don't actually check the value of the boolean we can make this a bit more efficient: 
```suggestion
func rewardSetToMap(rewardSet []string) map[string]struct{} {
	rewardMap := make(map[string]struct{})
	for _, v := range rewardSet {
		rewardMap[v] = struct{}{}
	}

	return rewardMap
}
```